yt/frontends/boxlib/data_structures.py

```diff
@@ -1405,6 +1403,16 @@ def _read_header(raw_file, field):
                 lo_corner, hi_corner, node_type = _line_to_numpy_arrays(clean_line)
                 boxes.append((lo_corner, hi_corner, node_type))
 
+            try:
+                # nghost_line is a single number
+                ng = int(nghost_line[0])
+                ndims = len(lo_corner)
+                nghost = np.array(ndims * [ng])
+            except ValueError:
+                # nghost_line is (#,#,#)
+                nghost_list = nghost_line[0][1:-1].split(",")
```

Comment:
took me a little while to understand why we'd want to ignore the first and last characters, I would refactor this line as 
```suggestion
                nghost_list = nghost_line[0].strip("()").split(",")
```